### PR TITLE
Proofread and do some enhancements

### PR DESCRIPTION
**Pull request by ShanlanLi**
Major changes:
1. Fix some format mistakes, like:
image
2. Check and resolve the issue of some code highlighting not being displayed with the help of @luotianchun 
image

4. Standardize the style of inconsistent headings to align with the majority of headings, including capitalization and punctuation.
image

5. Made some minor changes to languages where it may sound a bit strange.

image

6. Other small enhancements.
image

### CODE REVIEW

File: docs/advanced/schema.md [hunk 1]
```diff
@@ -10,13 +10,13 @@ RisingWave itself is a stream processing system and a database. Therefore, it no
...
 
-### Adding a column:
+### Adding a Column
 
 To avoid data loss with the newly added column, users need to first add the column to `table_rw` and optionally set its default value for all historical data. Then, after adding the column to `table_pg`, new data can be imported.
 
 Existing materialized views in RisingWave, such as `mv_rw`, and other stream tasks, will not automatically update to reference this newly added column. Users must create new materialized views or stream tasks to reference it. In the future, RisingWave plans to support online query changes for materialized views, giving users the opportunity to avoid rebuilding these stream tasks.
 
-### Dropping a column:
+### Dropping a column
```
Comment (CharlieSYH) [inline docs/advanced/schema.md]: Since we're using title cases in the Tutorials, capitalize H here.

**Review by CharlieSYH — APPROVED** [review]

**Comment by hengm3467** [comment]
Good work! One minor change we want to make is the capitalization in nodes. "Compute Node" is not necessary. Just treat it as common words (compute node for most cases, and Compute node if starting a new sentence). Thanks.

**Review by hengm3467 — APPROVED** [review]
Otherwise LGTM. Thanks!

**Review by WanYixian — APPROVED** [review]
LGTM

**Pull request merged**